### ISSUE TITLE
odbc@beta install failing

### ISSUE THREAD

**Issue by ghost** [wankdanker/node-odbc#65]
`
error /Users/qlecler/courtier_ph/node_modules/odbc: Command failed.
Exit code: 1
Command: node-gyp configure build
Arguments: 
Directory: /Users/qlecler/courtier_ph/node_modules/odbc
Output:
gyp info it worked if it ends with ok
gyp info using node-gyp@3.8.0
gyp info using node@10.15.3 | darwin | x64
gyp info spawn /usr/bin/python
gyp info spawn args [ '/Users/qlecler/.nvm/versions/node/v10.15.3/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py',
gyp info spawn args   'binding.gyp',
gyp info spawn args   '-f',
gyp info spawn args   'make',
gyp info spawn args   '-I',
gyp info spawn args   '/Users/qlecler/courtier_ph/node_modules/odbc/build/config.gypi',
gyp info spawn args   '-I',
gyp info spawn args   '/Users/qlecler/.nvm/versions/node/v10.15.3/lib/node_modules/npm/node_modules/node-gyp/addon.gypi',
gyp info spawn args   '-I',
gyp info spawn args   '/Users/qlecler/.node-gyp/10.15.3/include/node/common.gypi',
gyp info spawn args   '-Dlibrary=shared_library',
gyp info spawn args   '-Dvisibility=default',
gyp info spawn args   '-Dnode_root_dir=/Users/qlecler/.node-gyp/10.15.3',
gyp info spawn args   '-Dnode_gyp_dir=/Users/qlecler/.nvm/versions/node/v10.15.3/lib/node_modules/npm/node_modules/node-gyp',
gyp info spawn args   '-Dnode_lib_file=/Users/qlecler/.node-gyp/10.15.3/<(target_arch)/node.lib',
gyp info spawn args   '-Dmodule_root_dir=/Users/qlecler/courtier_ph/node_modules/odbc',
gyp info spawn args   '-Dnode_engine=v8',
gyp info spawn args   '--depth=.',
gyp info spawn args   '--no-parallel',
gyp info spawn args   '--generator-output',
gyp info spawn args   'build',
gyp info spawn args   '-Goutput_dir=.' ]
gyp info spawn make
gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
  CXX(target) Release/obj.target/odbc/src/odbc.o
In file included from ../src/odbc.cpp:18:
/Users/qlecler/courtier_ph/node_modules/node-addon-api/napi.h:30:6: error: Exception support not detected.       Define either NAPI_CPP_EXCEPTIONS or NAPI_DISABLE_CPP_EXCEPTIONS.
    #error Exception support not detected. \
     ^
../src/odbc.cpp:758:15: warning: unused variable 'params' [-Wunused-variable]
  Napi::Array params = ODBC::ParametersToArray(env, data);
              ^
1 warning and 1 error generated.
make: *** [Release/obj.target/odbc/src/odbc.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/Users/qlecler/.nvm/versions/node/v10.15.3/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:262:23)
gyp ERR! stack     at ChildProcess.emit (events.js:189:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:248:12)
gyp ERR! System Darwin 18.6.0
gyp ERR! command "/Users/qlecler/.nvm/versions/node/v10.15.3/bin/node" "/Users/qlecler/.nvm/versions/node/v10.15.3/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "configure" "build"
gyp ERR! cwd /Users/qlecler/courtier_ph/node_modules/odbc
gyp ERR! node -v v10.15.3
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok
`

**Comment by markirish** [comment]
This is a known issue on Windows, and is actually fixed in the branch `v2.0`. I will hopefully have time today to publish a bunch of new changes to npm.

If you want to install it now, you can run

```
npm install git://github.com/wankdanker/node-odbc.git#v2.0
```

**Comment by markirish** [comment]
Was fixed on the official release of odbc v2.0, installation on windows should now work with:

```
npm install odbc
```
